### PR TITLE
Display schedule time relative to now.

### PR DESCRIPTION
**Pull request by morgoth**
Uses https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/DateHelper.html#method-i-time_ago_in_words

### CODE REVIEW

File: engine/app/helpers/good_job/application_helper.rb [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 module GoodJob
   module ApplicationHelper
+    def relative_time(timestamp)
+      text = timestamp.future? ? "in #{time_ago_in_words(timestamp)}" : "#{time_ago_in_words(timestamp)} ago"
+      content_tag(:time, text, datetime: timestamp, title: timestamp)
```
Comment (bensheldon) [inline engine/app/helpers/good_job/application_helper.rb:6]: I think `content_tag` is deprecated in favor of `tag.time`.

**Review by bensheldon — APPROVED** [review]
Big improvement. I'm a fan of relative time with timestamp title. 

You might know better than me about the future compatibility of `content_tag`.

File: engine/app/helpers/good_job/application_helper.rb (hunk 1 above)
Comment (morgoth) [reply engine/app/helpers/good_job/application_helper.rb]: I changed it. I wasn't sure in which version of Rails `tag` was introduced, but we're already using it in engine, so it is fine.

**Comment by bensheldon** [comment]
😍
 
Screen Shot 2021-10-01 at 5 47 41 PM

**Pull request merged**